app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughFinalFragmentTest.kt

```diff
@@ -189,7 +189,7 @@ class WalkthroughFinalFragmentTest {
   }
 
   @Test
-  fun testWalkthroughFinalFragment_yesNoButton_isDisplayed() {
+  fun testWalkthroughFinalFragment_yesNoBtn_isDisplayed() {
```

Comment:
Is displayed should be together with yes no button as suggested in last comment.

Reply:
done